index.html

```diff
@@ -22,33 +22,49 @@
       width: 80%;
       max-width: 320px;
     }
-
-    .app__init-logo {
+    .app__init-left-face-logo {
+      position: absolute;
       margin: 0 auto;
       width: 100%;
+      animation: init-left-face-logo .6s ease-out infinite;
     }
-
-    .app__init-load-spinner {
-      width: 60px;
-      height: 60px;
+    .app__init-top-face-logo {
+      position: absolute;
       margin: 0 auto;
+      width: 100%;
+      animation: init-top-face-logo .6s ease-out infinite;
     }
-
-    .app__init-load-spinner:after {
-      content: "";
-      display: block;
-      width: 40px;
-      height: 40px;
-      border-radius: 50%;
-      border: 8px solid;
-      border-color: #3a4180 transparent;
-      animation: init-load-spinner 1.2s linear infinite;
+    .app__init-right-face-logo {
+      position: absolute;
+      margin: 0 auto;
+      width: 100%;
+      animation: init-right-face-logo .6s ease-out infinite;
+    }
+    .app__init-logo-text {
+      position: absolute;
+      margin: 0 auto;
+      width: 100%;
     }
 
-    @keyframes init-load-spinner {
-      0% { transform: rotate(0deg); }
-      100% { transform: rotate(360deg); }
+    @keyframes init-left-face-logo {
+      0% { opacity: 0 }
+      33% { opacity: 1 }
+      66% { opacity: 0 }
+      100% { opacity: 0 }
+    }
+    @keyframes init-top-face-logo {
+      0% { opacity: 0 }
+      33% { opacity: 0 }
+      66% { opacity: 1 }
+      100% { opacity: 0 }
     }
+    @keyframes init-right-face-logo {
+      0% { opacity: 1 }
+      33% { opacity: 0 }
+      66% { opacity: 0 }
+      100% { opacity: 1 }
+    }
```

Comment:
merge three animations into a single animations with nesting to prevent outing of sync, like
```css
@keyframes anim {
  0% {
    .top-face: {}
    .right-face: {}
    .left-face: {}
  }
  33%: {
  /* ... */
  }
}
```